content/usgs-packages/Application_Intro.Rmd

```diff
@@ -32,14 +32,16 @@ set.seed(1)
 
 ## Lesson Summary
 
-TBD
+This lesson will revisit the packages taught in previous lessons and use some of their functions to create a common data analysis workflow in a modular and reproducible manner. Then, you can learn how a modular approach compares to traditional analyses.
```

Comment:
i mean that you're promising to compare modular to traditional. are you going to show a script that implements a geoknife extraction/downloading process using base R only? sounds scary to me. i really just made this comment to make sure you were thinking about it - i'm content to wait until i see the rest of the lesson before you make any particular edits here.